### ISSUE TITLE
[Feature Request] field to link contacts to tasks, similar to tags

### ISSUE THREAD

**Issue by mstyp** [nextcloud/tasks#2019]
This is different then the "assign users to a task" feature request, since there are a bunch of people in my contacts book that I would like to "tag", but who don't have accounts. 

If I have a task that says "email person x" or "call person x", where person x is in the nextcloud contacts app, there should be a feild where I can start typing that persons nickname and then see it in a dropdown menue so I can quickly navigate to their contact page or sort tasks based on associated contacts.

**Comment by IgorA100** [comment]
Yes, I support this idea!
Very useful!

**Comment by polcape** [comment]
Yes I agree, no one caldav client have this feature but I think caldav tasks support CONTACT proprierties
